packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -0,0 +1,298 @@
+import { IStyleFunction } from '../../Utilities';
+import {
+  FontSizes,
+  FontWeights,
+  HighContrastSelector,
+  IRawStyle,
+  IStyle,
+  getGlobalClassNames,
+  normalize
+} from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-Dropdown-container',
+  label: 'ms-Dropdown-label',
+  dropdown: 'ms-Dropdown',
+  title: 'ms-Dropdown-title',
+  caretDownWrapper: 'ms-Dropdown-caretDownWrapper',
+  caretDown: 'ms-Dropdown-caretDown',
+  callout: 'ms-Dropdown-callout',
+  panel: 'ms-Dropdown-panel',
+  dropdownItems: 'ms-Dropdown-items',
+  dropdownDivider: 'ms-Dropdown-divider',
+  dropdownOptionText: 'ms-Dropdown-optionText',
+  dropdownItemHeader: 'ms-Dropdown-header',
+  titleIsPlaceHolder: 'ms-Dropdown-titleIsPlaceHolder',
+  titleHasError: 'ms-Dropdown-title--hasError'
+};
+
+const DROPDOWN_HEIGHT = 32;
+const DROPDOWN_ITEMHEIGHT = 32;
+
+const highContrastAdjustMixin = {
+  // highContrastAdjust mixin
+  '@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: black-on-white)': {
+    MsHighContrastAdjust: 'none'
+  }
+};
+
+const highContrastItemAndTitleStateMixin: IRawStyle = {
+  selectors: {
+    [HighContrastSelector]: {
+      backgroundColor: 'Highlight',
+      borderColor: 'Highlight',
+      color: 'Highlight'
+    },
+    ...highContrastAdjustMixin
+  }
+};
+
+const highContrastBorderState: IRawStyle = {
+  selectors: {
+    [HighContrastSelector]: {
+      borderColor: 'Highlight'
+    }
+  }
+};
+
+export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = props => {
+  const {
+    theme,
+    hasError,
+    className,
+    isOpen,
+    disabled,
+    required,
+    isRenderingPlaceholder,
+    panelClassName,
+    calloutClassName
+  } = props;
+
+  if (!theme) {
+    throw new Error('theme is undefined or null in base Dropdown getStyles function.');
+  }
+
+  const globalClassnames = getGlobalClassNames(GlobalClassNames, theme);
+  const { palette, semanticColors } = theme;
+
+  const rootHoverFocusActiveSelectorNeutralDarkMixin: IStyle = {
+    color: palette.neutralDark
+  };
+
+  const rootHoverFocusActiveSelectorNeutralSecondaryMixin: IStyle = {
+    color: palette.neutralSecondary
+  };
+
+  const borderColorError: IStyle = {
+    borderColor: semanticColors.errorText
+  };
+
+  const dropdownItemStyle: IStyle = [
+    'ms-Dropdown-item',
+    {
+      backgroundColor: 'transparent',
+      boxSizing: 'border-box',
+      cursor: 'pointer',
+      display: 'block',
+      padding: '4px 16px',
+      width: '100%',
+      minHeight: DROPDOWN_ITEMHEIGHT,
+      lineHeight: 20,
+      height: 'auto',
+      position: 'relative',
+      border: '1px solid transparent',
+      wordWrap: 'break-word',
+      overflowWrap: 'break-word',
+      textAlign: 'left'
+    }
+  ];
+
+  const dropdownItemSelected: IStyle = [
+    ...dropdownItemStyle,
+    {
+      backgroundColor: palette.neutralQuaternaryAlt,
```

Comment:
semanticColors.menuItemBackgroundHovered

Reply:
Even if the dropdown isn't in a hovered state right here?